recipes/acado/all/conanfile.py

```diff
@@ -0,0 +1,116 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AcadoConan(ConanFile):
+    name = "acado"
+    description = "ACADO Toolkit is a software environment and algorithm collection for automatic control and dynamic optimization."
+    license = "LGPL-3.0"
+    topics = ("conan", "acado", "control", "optimization", "mpc")
+    homepage = "https://github.com/acado/acado"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "cmake/qpoases.cmake", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "codegen_only": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "codegen_only": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
```suggestion
            del self.options.fPIC
        if self.settings.compiler == "Visual Studio" and \
           "MT" in self.settings.compiler.runtime and self.options.shared:
            raise ConanInvalidConfiguration("Acado does not support MT runtime + shared library.")
```

It's an optional profile, which many doesn't support. We can skip it to avoid that error.